src/AppRoutes.tsx

```diff
@@ -16,82 +14,91 @@ const EditRegistration = lazy(() => import('./pages/registration/new_registratio
 const CreateProject = lazy(() => import('./pages/project/project_wizard/CreateProject'));
 const EditProject = lazy(() => import('./pages/project/project_wizard/EditProject'));
 const PublicRegistration = lazy(() => import('./pages/public_registration/PublicRegistration'));
+const MyPagePage = lazy(() => import('./pages/my_page/MyPagePage'));
 const PrivacyPolicy = lazy(() => import('./pages/infopages/PrivacyPolicy'));
-const ProjectsPage = lazy(() => import('./pages/projects/ProjectsPage'));
 const PublicResearchProfile = lazy(() => import('./pages/research_profile/PublicResearchProfile'));
-const MyPagePage = lazy(() => import('./pages/my_page/MyPagePage'));
 const TasksPage = lazy(() => import('./pages/messages/TasksPage'));
+const SignedOutPage = lazy(() => import('./pages/infopages/SignedOutPage'));
+const ProjectsPage = lazy(() => import('./pages/projects/ProjectsPage'));
 const Logout = lazy(() => import('./layout/Logout'));
 const LoginPage = lazy(() => import('./layout/LoginPage'));
-const SignedOutPage = lazy(() => import('./pages/infopages/SignedOutPage'));
+const NotFound = lazy(() => import('./pages/errorpages/NotFound'));
 
 export const AppRoutes = () => {
-  const { t } = useTranslation();
   const user = useSelector((store: RootState) => store.user);
 
   const isAuthenticated = !!user;
   const hasCustomerId = isAuthenticated && !!user.customerId;
   const isCreator = hasCustomerId && user.isCreator;
   const isCurator = hasCuratorRole(user);
   const isEditor = hasCustomerId && user.isEditor;
-  const isAdmin = hasCustomerId && (user.isAppAdmin || user.isInstitutionAdmin || user.isInternalImporter);
-  const isNviCurator = hasCustomerId && user.isNviCurator;
+  const canSeeBasicData = hasCustomerId && (user.isAppAdmin || user.isInstitutionAdmin || user.isInternalImporter);
+  const isNviCurator = !!user?.isNviCurator;
 
   return (
-    <Suspense fallback={<PageSpinner aria-label={t('common.page_title')} />}>
-      <Switch>
-        <Route
-          exact
-          path={[
-            UrlPathTemplate.Home,
-            UrlPathTemplate.Search,
-            UrlPathTemplate.Reports,
-            UrlPathTemplate.ReportsNvi,
-            UrlPathTemplate.ReportsInternationalCooperation,
-            UrlPathTemplate.ReportsClinicalTreatmentStudies,
-          ]}
-          component={Dashboard}
-        />
+    <Routes>
+      <Route element={<Layout />}>
+        <Route path={UrlPathTemplate.Root} element={<Dashboard />}>
```

Comment:
Dette forårsaker feilen jeg har kommentert om i Jira: https://sikt.atlassian.net/browse/NP-47603?focusedCommentId=110942